packages/cli/README.md

```diff
@@ -57,7 +57,7 @@ USAGE
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key          DID Key
+  -k, --key=key          Private DID Key
```

Comment:
I thought we had decided we would remove the docs for the `--key` arg from the commands that it doesn't make sense for?

Reply:
I thought we had decided to keep it because it's baked in pretty deep and removing it would be difficult. If a user provides a key it doesn't break or alter the commands at all.

Reply:
My understanding was that we weren't going to remove the arg entirely because it's baked in too deep, but we'd remove the documentation for it from the commands for which it doesn't make sense or do anything.

Reply:
Right, I remember now. The plan was to remove it from the readme documentation and that's all. Easy fix that I'll get in tomorrow.

Reply:
Can we also remove it from the per-command docs and help text output?  No reason to tell users about a flag that isn't going to do anything.  Just more likely to lead to confusion

Reply:
Unfortunately, that is more complex and pretty hard-baked into the core itself. I can _try_ but I would much rather not. All commands inherit the flag from our base command (`packages/cli/src/command.ts`). I can try to remove it, but again I'd much rather not.

Reply:
I don't understand.  This comment thread is literally on the line of the source code with the comments isn't it?  Can't we just remove literally the line I commented on, and the equivalent lines from other commands?

Reply:
Sorry, I might have been confused,

We can absolutely remove it from the commands listed here in the README. Not a problem at all.

Removing it from the commands help is difficult because they are declared in the parent command and not in any of the child commands. We extend oclif's command & add our own details to it. This is the file that is adding the key flag to all commands https://github.com/ceramicstudio/js-glaze/blob/feat/merge-ceramic-cli/packages/cli/src/command.ts#L36.

Reply:
oh I'm sorry, this confusion is totally my fault.  Somehow I missed that this comment thread is on the readme, not the commands themselves.  I do think we should remove it from the readme though, might as well (but leave it for commands that do writes, namely the `create` and `update` commands)